tests/unit/style/stylesheets.rs

```diff
@@ -292,14 +293,19 @@ impl CSSInvalidErrorReporterTest {
 }
 
 impl ParseErrorReporter for CSSInvalidErrorReporterTest {
-    fn report_error(&self, input: &mut CssParser, position: SourcePosition, message: &str) {
+    fn report_error(&self, input: &mut CssParser, position: SourcePosition, message: &str,
+        servo_url: &ServoUrl) {
+
+        let url = servo_url.clone();
```

Comment:
let's remove this line and add `.clone()` to `url` when creating the `CSSError` instance